go-controller/pkg/util/iptables.go

```diff
@@ -12,14 +12,16 @@ import (
 // IPTablesHelper is an interface that wraps go-iptables to allow
 // mock implementations for unit testing
 type IPTablesHelper interface {
+	List(string, string) ([]string, error)
```

Comment:
doc text missing

Reply:
maybe rename this to ListRules ?

Reply:
Will rectify

Reply:
Actually, we have `Delete` / `Insert` / `Exists` will all refer to iptables rules, for consistency I think it's better to leave it as `List` or modify all of them to `*Rules`. I will do the latter

Reply:
But that will mean an even bigger PR....do we _really_ want that? :smiley:

Reply:
Actually, nevermind: @girishmg already merged a patch for this: https://github.com/ovn-org/ovn-kubernetes/pull/1347/files